awx/main/tests/functional/api/test_inventory.py

```diff
@@ -609,23 +609,9 @@ def test_get_constructed_inventory(self, constructed_inventory, admin_user, get)
         r = get(url=reverse('api:constructed_inventory_detail', kwargs={'pk': constructed_inventory.pk}), user=admin_user, expect=200)
         assert r.data['update_cache_timeout'] == 53
 
-    def test_patch_constructed_inventory(self, constructed_inventory, admin_user, patch):
+    def test_patch_constructed_inventory_generated_source_limits_editable_fields(self, constructed_inventory, admin_user, project, patch, inventory):
```

Comment:
For the unused arguments, inject those fixtures differently:
```suggestion
    @pytest.mark.usefixtures('inventory', 'project')
    def test_patch_constructed_inventory_generated_source_limits_editable_fields(self, constructed_inventory, admin_user, patch) -> None:
```